Library/Homebrew/list.sh

```diff
@@ -35,7 +36,9 @@ homebrew-list() {
   if [[ -t 1 ]]
   then
     tty=1
+    ls_args+=("-Cq")
```

Comment:
Sadly this happens too late now so overrides args like `-1`

Reply:
Fortunately, this block can easily be lifted up. I've done so now.